src/extensions/extension-loader/extension-loader.ts

```diff
@@ -324,11 +324,13 @@ export class ExtensionLoader {
             return {
               extId,
               instance,
+              installedExtension: extension,
```

Comment:
I don't think we need this, we can always just do `instance.manifest`

Reply:
That's slightly different type, and has a bit less information. I think we should keep installedExtension in the error log, and that would match the original functionality too.

Reply:
Good point, removed extId and isBudled from the object.